ui/src/app/components/workspace/workspace.component.ts

```diff
@@ -301,7 +304,25 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     return false
   }
   prepareMigration() {
-    this.router.navigate(['/prepare-migration'])
+    this.fetch.getErrors().subscribe({
+      next: (res: any) => {
+        if (res != null && res.length !=0)
+        {
+          let errMsg = 'Please fix the errors for the following tables to move ahead: '+ res.join(',')
```

Comment:
Can you add a screenshot of how this looks please? 
For a large number of tables this might go out of bounds of the dialog box width of 500 px. 🤔